README.md

```diff
@@ -84,7 +85,10 @@ chart-dirs:
 build-id: pr-42
 ```
 
-`ct install --config config.yaml`
+Config usage:
```

Comment:
For consistency change this to:

```suggestion
#### Config usage:
```

Reply:
Title case, please: `Config Usage`

Reply:
Nice catch - updated suggestion (my mind always looks for `Sentence case` 🤷‍♂️ )